### PR TITLE
Add Python packages to the manifest

### PR DESCRIPTION
**Pull request by cwharris**

### CODE REVIEW

**Comment by trxcllnt** [comment]
* need to update the `devcontainer-features.json` version
* does this PR add the python package names to `manifest.yaml`?

File: features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml [hunk 1]
```diff
@@ -62,7 +106,12 @@ repos:
   cpp:
     - name: cuml
       sub_dir: cpp
-      depends: [rmm, raft, cumlprims_mg]
+      depends: [fmt, rmm, raft, cumlprims_mg]
+      args: "-DSINGLEGPU=OFF -DDETECT_CONDA_ENV=OFF -DBUILD_CUML_C_LIBRARY=OFF -DBUILD_CUML_CPP_LIBRARY=ON -DBUILD_CUML_EXAMPLES=OFF -DBUILD_CUML_TESTS=OFF -DBUILD_PRIMS_TESTS=OFF -DBUILD_CUML_BENCH=OFF -DBUILD_CUML_PRIMS_BENCH=OFF -DCUML_RAFT_CLONE_ON_PIN=OFF -DDISABLE_DEPRECATION_WARNINGS=ON"
```
Comment (cwharris) [inline features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml:110]: I tried doing `args: |`, but I think it had trouble with that. I should try again.

Comment (cwharris) [reply features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml:113]: yeah, it ends up interpolating to:
```bash
cmake_args+=(-DSINGLEGPU=OFF;local -DDETECT_CONDA_ENV=OFF;local -DBUILD_CUML_C_LIBRARY=OFF;local -DBUILD_CUML_CPP_LIBRARY=ON;local -DBUILD_CUML_EXAMPLES=OFF;local -DBUILD_CUML_TESTS=OFF;local -DBUILD_PRIMS_TESTS=OFF;local -DBUILD_CUML_BENCH=OFF;local -DBUILD_CUML_PRIMS_BENCH=OFF;local -DCUML_RAFT_CLONE_ON_PIN=OFF;local -DDISABLE_DEPRECATION_WARNINGS=ON;local );
```

Comment (cwharris) [reply features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml]: fixed.

**Pull request merged**